Master -> Main Rename

dannylamb opened the pull request:
Part of https://github.com/Islandora/documentation/issues/1595

rosiel committed:
Render the example helpful tip properly. (#1586)

* Render the example helpful tip properly.

* Fix note syntax, again.

rosiel committed:
Mention the User Call (#1587)

elizoller committed:
Update rdf-mapping.md doc (#1585)

dannylamb committed:
Update committers.md (#1592)

jonathangreen committed:
Fix committer nomination email address (#1593)

* Fix old email address.

* Update closeCommitterVote.txt

* De-CLAW

* De-CLAW

* DeCLAW

* Change email

* De-CLAW

jonathangreen committed:
Update travis for the main branch switch (#1594)

dannylamb committed:
Master -> Main Rename

rosiel commented on CONTRIBUTING.md:
```diff
@@ -12,7 +12,7 @@ If you would like to submit a use case to the Islandora 8 project, please submit
 ### Documentation
 
-You can contribute documentation in two different ways. One way is to create an issue [here](https://github.com/Islandora/documentation/issues/new), prepending "Documentation:" to the title of the issue. Another way is by pull request, which is the same process as [Contribute Code](https://github.com/Islandora/documentation/blob/master/CONTRIBUTING.md#contribute-code). All documentation resides i ...
+You can contribute documentation in two different ways. One way is to create an issue [here](https://github.com/Islandora/documentation/issues/new), prepending "Documentation:" to the title of the issue. Another way is by pull request, which is the same process as [Contribute Code](https://github.com/Islandora/documentation/blob/mainONTRIBUTING.md#contribute-code). All documentation resides in [` ...
```
Oooooh this didn't go so well. But I like "mamains".

DonRichards committed:
Remove comments about WebAC (#1555)

* Remove comments about WebAC

WebAC is not used but is listed as a feature added when using Fedora. Although it is there, I'd suggest leaving off of this documentation.

* Update user-intro.md

rosiel committed:
Add more details about starting up an islandora. (#1597)

* Add more details about starting up an islandora.

* Update index.md

* Mention use of dev branch for prebuilt base box

* Update index.md

* Fix typo in VBE

* Make clearer the different levels of sandbox reqs.

whikloj commented on CONTRIBUTING.md:
```diff
@@ -58,9 +58,9 @@ Contributions to the Islandora codebase should be sent as GitHub pull requests.
 Take a look at [Creating a pull request](https://help.github.com/articles/creating-a-pull-request). In a nutshell you need to:
 
-1. [Fork](https://help.github.com/articles/fork-a-repo) this repository to your personal or institutional GitHub account (depending on the CLA you are working under). Be cautious of which branches you work from though (you'll want to base your work off master, or for Drupal modules use the most recent version branch). See [Fork a repo](https://help.github.com/articles/fork-a-repo) for detailed in ...
+1. [Fork](https://help.github.com/articles/fork-a-repo) this repository to your personal or institutional GitHub account (depending on the CLA you are working under). Be cautious of which branches you work from though (you'll want to base your work off mainor for Drupal modules use the most recent version branch). See [Fork a repo](https://help.github.com/articles/fork-a-repo) for detailed instru ...
```
script failure `master, or` -> `mainor`

whikloj commented on CONTRIBUTING.md:
```diff
@@ -58,9 +58,9 @@ Contributions to the Islandora codebase should be sent as GitHub pull requests.
+1. [Fork](https://help.github.com/articles/fork-a-repo) this repository to your personal or institutional GitHub account (depending on the CLA you are working under). Be cautious of which branches you work from though (you'll want to base your work off mainor for Drupal modules use the most recent version branch). See [Fork a repo](https://help.github.com/articles/fork-a-repo) for detailed instru ...
 2. Commit any changes to your fork.
-3. Send a [pull request](https://help.github.com/articles/creating-a-pull-request) using the [pull request template](https://github.com/Islandora/documentation/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to the Islandora GitHub repository that you forked in step 1.  If your pull request is related to an existing issue -- for instance, because you reported a [bug/issue](https://github.com/Island ...
+3. Send a [pull request](https://help.github.com/articles/creating-a-pull-request) using the [pull request template](https://github.com/Islandora/documentation/blob/maingithub/PULL_REQUEST_TEMPLATE.md) to the Islandora GitHub repository that you forked in step 1.  If your pull request is related to an existing issue -- for instance, because you reported a [bug/issue](https://github.com/Islandora/ ...
```
Script failure `/blob/master/.github` -> `/blob/maingithub`

whikloj commented on docs/technical-documentation/flysystem.md:
```diff
@@ -14,13 +14,13 @@ The Drupal Flysystem module uses [flysystem stream wrappers](https://github.com/
 ### The Plugin and Adapter
 
-Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between th ...
+Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between th ...
```
This looks to have been there before but the URL for the `Chullo Fedora API` is missing `https:/` and so the link is broken.

Also the commented out text at the bottom states
> After https://github.com/Islandora-CLAW/islandora/pull/119 is merged:

It is merged, so this can (I assume) by revealed.

Also there were some CLAWs in here.
 ```suggestion
Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between them. The adapter interacts with Fedora using an instance of the [Chullo Fedora API](https://github.com/Islandora/chullo/blob/main/src/IFedoraApi.php) it receives from the Drupal Flysystem plugin. The Drupal Flysystem plugin's main responsibility is to instantiate the Chullo Fedora API object with the proper authentication and pass it to the Flysystem adapter. To authenticate with Fedora the plugin adds a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html) to the Chullo's [Guzzle](http://docs.guzzlephp.org) client which adds a [JWT](https://jwt.io/) authentication header to each request. To authenticate with Fedora the plugin adds a handler to the Chullo's Guzzle client with a copy of the site's JWT authentication object so that each Fedora request can generate its own Bearer token.
```

rosiel replied:
requiring the full de-clawing of the documentation right now would also require re-writing and also updating a large number of formerly-useful pages... that's an ongoing process, can we do the master-main rename first?

whikloj replied:
I have no problem not doing a full de-clawing, I just noted the changes on this line as the PR already touched it.

rosiel replied:
phew. ty. Yeah there are a number of pages in here that need love. I'm currently looking at "hacking on islandora".

whikloj commented on docs/technical-documentation/testing-notes.md:
```diff
@@ -2,7 +2,7 @@
 The map between Drupal URIs and their corresponding Fedora URIs is stored in a service called Gemini. An Islandora "pseudo-field" can be enabled that will display the corresponding Fedora URI on the page of a node, taxonomy term, or media object.  You can also query Gemini directly. Instructions for doing that are provided below.
 
-For more information about the Gemini service, see the [Gemini README](https://github.com/Islandora-CLAW/Crayfish/tree/master/Gemini).
+For more information about the Gemini service, see the [Gemini README](https://github.com/Islandora-CLAW/Crayfish/tree/main/Gemini).
```
CLAW in need of de-ing.

rosiel reviewed (commented):
oops - i caught similar things to @whikloj

manez committed:
Update CONTRIBUTING.md (#1601)

jonathangreen committed:
Add Jordan to committers list (#1602)

jonathangreen commented:
Something I noticed yesterday when I made #1602 is that we have some PR going into master right now and some into main, so the two branches are diverging. It looks like #1601 went in against master. And this PR is against master.

jonathangreen commented:
It seems like master needs to be merged into main and then deleted, as @whikloj noted in https://github.com/Islandora-Devops/ansible-role-grok/pull/9, so they don't keep getting out of sync. Just wanted to note it here so that it doesn't get lost on this PR.

ysuarez committed:
Migration docs updates (#1603)

* Added link to newer migration Youtube vid, added migration-overview.md

* Replaced used of "*" with "-" for bulleted lists

* Updated headings in docs and spelling changes

rosiel committed:
Main update (#1612)

* Render the example helpful tip properly. (#1586)

* Render the example helpful tip properly.

* Fix note syntax, again.

* Update rdf-mapping.md doc (#1585)

Co-authored-by: Eli Zoller <5439169+elizoller@users.noreply.github.com>

rosiel committed:
Merge changes to add @manez to committers. (#1613)

* Merge changes to add @manez to committers.

* Alphabeticalize committer names.

Co-authored-by: dannylamb <dlamb@islandora.ca>

dannylamb commented:
_rolls up his sleeves and gets to it_ :man_mechanic:

dannylamb committed:
De-CLAWing

dannylamb committed:
Derp

dannylamb commented:
eh... that rebase half worked....  let me see...

dannylamb committed:
Merge branch 'master' of https://github.com/Islandora/documentation into master-main-rename

dannylamb committed:
Merge branch 'main' of https://github.com/Islandora/documentation into master-main-rename

dannylamb committed:
Missing parenthesis

dannylamb commented:
Ok, this has `master` and `main` pulled in.  Should be ready now.  Please squash and merge.

dannylamb commented:
You'll notice a little extra de-CLAWing :tophat:

rosiel commented:
ping @whikloj - Github's blocking, pending your change request.

dannylamb commented:
@whikloj, would you mind releasing your hold on this one?

dannylamb commented:
@whikloj bump
